src/content/en/updates/2018/09/inside-browser-part1.md

```diff
@@ -0,0 +1,298 @@
+  </figcaption>
+</figure>
+
+
+## Browser Architecture {: #browser-architecture }
+
+So how is a web browser built using processes and threads? Well, it could be one process with many 
+different threads or many different processes with a few threads communicating over IPC. 
+<figure>
+  <img src="../../images/inside-browser/browser-arch.png" alt="browser architecture">
+  <figcaption>
+   Figure 7: Different browser architectures in process/thread diagram
+  </figcaption>
+</figure>
+
+The important thing to note here is that these different architectures are implementation details. 
+There is no standard specification on how one might build a web browser. One browser’s approach may 
+be completely different from another. 
+
+For the sake of this blog series, we are going to use Chrome’s recent architecture described in the 
+diagram below.
+
+At the top is the browser process coordinating with other processes that take care of different 
+parts of the application. For the renderer process, multiple processes are created and assigned to 
+each tab. Until very recently, Chrome gave each tab a process when it could; now it tries to give 
+each site its own process, including iframes (see [Site Isolation](#site-isolation)).
+
+<figure>
+  <img src="../../images/inside-browser/browser-arch2.png" alt="browser architecture">
+  <figcaption>
+   Figure 8: Diagram of Chrome’s multi-process architecture. Multiple layers are shown under 
+   Renderer Process to represent Chrome running multiple Renderer Processes for each tab.
+  </figcaption>
+</figure>
+
+## Which process controls what?
+The following table describes each Chrome process and what it controls:
+
+<table>
+  <tr>
+    <th>Process</th>
+    <th>What it controls</th>
+  </tr>
+  <tr>
+    <td>Browser</td>
+    <td>
+      Controls "chrome" part of the application including address bar, bookmarks, back and 
+      forward buttons. <br>Also handles the invisible, privileged parts of a web browser such as 
+      network requests and file access.
+    </td>
+  </tr>
+  <tr>
+    <td>Renderer</td>
+    <td>Controls anything inside of the tab where a website is displayed.</td>
+  </tr>
+  <tr>
+    <td>Plugin</td>
+    <td>Controls any plugins used by the website, for example, flash.</td>
+  </tr>
+  <tr>
+    <td>GPU</td>
+    <td>
+      Handles GPU tasks in isolation from other processes. It is separated into different process 
+      because GPUs handles requests from multiple apps and draw them in the same surface.
+    </td>
+  </tr>
+</table>
+
+<figure>
+  <img src="/web/updates/images/inside-browser/browserui.png" alt="Chrome processes">
+  <figcaption>
+   Figure 9: Different processes pointing to different parts of browser UI
+  </figcaption>
+</figure>
+
+There are even more processes like the Extension process and utility processes. If you want to see 
+how many processes are running in your Chrome, click the options menu icon︙at the top right 
```

Comment:
Consider using the [material icon](https://developers.google.com/internal/publishing/reference/styles#material-icons) (more_vert) for the three dot menu. At a minimum, add the appropriate spacing between the character